### DISCUSSION TITLE
Purging Google Services

### DISCUSSION THREAD

**Discussion by munawarcmd** [Alex313031/thorium#440] [General]
After observation, we've realized Thorium has big potentials and edge over other browsers primarily due to speed and efficiency from recent benchmarks. We would love to hear Thorium going beyond its capabilities by eliminating possible malicious baked in services by Google just like Ungoogled-Chromium have done past making a complete privacy browser. This proposal could attract die hard FOSS fans whose believes in transparency and privacy. 

Since ungoogled chromium project is open source it is much easier for your team to follow their approach and implement without tough manual effort. This is part of my suggestions and removing google services could make a big difference to your user base near future. 

Good regards.

**Comment by haiderCho** [comment]
Aside from speed and efficiency another major selling point for Thorium is Google Sync. It's a complete replacement for Google Chrome. Ungoogled-Chromium even though it's very privacy focused, It's not for everyone. I replaced Google Chrome on many of my family and friend's PC who aren't very technical about browsing internet, Google Sync is crucial for them, their browser setup is always very easy with Thorium. Settings and Extensions are carried over even if I'm not there to set it up for them. I can just give them the link to Thorium to download and with a single SignIn their bookmarks, extensions are setup.

Someone should make a forked version of Ungoogled-Thorium but not at the cost of abandoning/replacing the current Thorium Project.
